src/Native/Unix/configure.cmake

```diff
@@ -87,6 +87,20 @@ check_c_source_compiles(
     "
     HAVE_FLOCK64)
 
+check_symbol_exists(
```

Comment:
You also need to add the new symbols to https://github.com/dotnet/corefx/blob/master/src/Native/Unix/Common/pal_config.h.in as `#cmakedefine01`